tests/admin_changelist/test_date_hierarchy.py

```diff
@@ -95,3 +95,11 @@ def test_invalid_params(self):
                 self.assertRaises(IncorrectLookupParameters),
             ):
                 self.assertDateParams(invalid_query, None, None)
+
+    @override_settings(ROOT_URLCONF="admin_changelist.urls")
+    def test_label_in_hierarchy(self):
+        self.client.force_login(self.superuser)
+        Event.objects.create(date=datetime(2017, 1, 1))
+        response = self.client.get("/admin/admin_changelist/event/")
```

Comment:
I see there’s other date_hierarchy tests elsewhere but this seems like the right place and right test. But - shouldn’t this be using `reverse` rather than hard-coded URL like other tests?